src/Backoffice/Flights/App/Controllers/UpdateFlightController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Flights\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Flights\App\Requests\UpsertFlightRequest;
+use Lightit\Backoffice\Flights\App\Resources\FlightResource;
+use Lightit\Backoffice\Flights\Domain\Actions\UpdateFlightAction;
+use Lightit\Backoffice\Flights\Domain\Models\Flight;
+
+class UpdateFlightController
+{
+    public function __invoke(
+        Flight $flight,
+        UpsertFlightRequest $upsertFlightRequest,
+        UpdateFlightAction $updateFlightAction,
+    ): JsonResponse {
+        $updatedFlight = $updateFlightAction->execute($flight, $upsertFlightRequest->toDto());
+
+        return FlightResource::make($updatedFlight)->response()->setStatusCode(JsonResponse::HTTP_OK);
```

Comment:
isn't the default status 200 already? no need to set the status, but its ok if you leave it